esp8266-arduino/ppd42ns-wificonfig-ppd-sds-dht/Readme.md

```diff
@@ -1,132 +1,135 @@
-Version für Sensoren PPD42NS, SDS011, DHT22, BMP180 und NEO-6M.  
+# Version für Sensoren PPD42NS, SDS011, DHT22, BMP180 und NEO-6M.
 
-Features:  
-- gleichzeitiger Betrieb mehrerer Sensoren  
-- Konfiguration über WLAN (Sensor als Access Point) möglich  
-- Unterstützung von OLED-Displays mit SSD1306  
-- Auswahl der API(s), an welche die Daten gesendet werden, inklusive der Möglichkeit, die Daten als CSV über USB auszugeben   
+Features:
+- gleichzeitiger Betrieb mehrerer Sensoren
+- Konfiguration über WLAN (Sensor als Access Point) möglich
+- Unterstützung von OLED-Displays mit SSD1306
+- Auswahl der API(s), an welche die Daten gesendet werden, inklusive der Möglichkeit, die Daten als CSV über USB auszugeben
 - nutzbar für ESP8266 und Arduino Feather M0 (LoRa)
 
 ToDo's:
 - TimeOut für WifiKonfig zum Laufen bekommen
 - MQTT-Konfig ergänzen
 
-Die grundsätzliche Konfiguration der Parameter erfolgt über die Datei ext_dev.h .
-  
-WLAN Konfiguration:
+Die grundsätzliche Konfiguration der Parameter erfolgt über die Datei `ext_dev.h`.
 
-(siehe auch Wiki-Seite auf Github [https://github.com/opendata-stuttgart/meta/wiki/Konfiguration-der-Sensoren](Konfiguration der Sensoren)  
+## WLAN Konfiguration
+
+(siehe auch Wiki-Seite auf Github [https://github.com/opendata-stuttgart/meta/wiki/Konfiguration-der-Sensoren](Konfiguration der Sensoren)
```

Comment:
- In `[]` die Beschreibung und in `()` die url
- `(` (1. Zeichen) wird nie geschlossen